### PR TITLE
Add day/night cycle tutorial

### LINKED ISSUE #3888: Add a tutorial for the day/night cycle
as there's currently no explanation

### PR DESCRIPTION
**Pull request by ender-titan1**
**Brief Description of What This PR Does**

This PR adds a tutorial for the day and night cycle. It shows after the player exits the editor in a patch with more than 0 ambient sunlight. When starting in a biome affected, it shows only after the first editor cycle, I don't know whether to change that or leave it as it is.

**Related Issues**

Closes #3888 

**Progress Checklist**

- [x] PR author has checked that this PR works as intended and doesn't
      break existing features:
      https://wiki.revolutionarygamesstudio.com/wiki/Testing_Checklist
      (this is important as to not waste the time of Thrive team
      members reviewing this PR)
- [x] Initial code review passed (this and further items should not be checked by the PR author)
- [x] Functionality is confirmed working by another person (see above checklist link)
- [x] Final code review is passed and code conforms to the 
      [styleguide](https://github.com/Revolutionary-Games/Thrive/blob/master/doc/style_guide.md).

Before merging all CI jobs should finish on this PR without errors, if
there are automatically detected style issues they should be fixed by
the PR author. Merging must follow our
[styleguide](https://github.com/Revolutionary-Games/Thrive/blob/master/doc/style_guide.md#git).

### CODE REVIEW

File: locale/en.po
Comment (Kasterisk) [inline locale/en.po:6051]: Accidental change.

File: src/microbe_stage/MicrobeStage.cs
Comment (Kasterisk) [inline src/microbe_stage/MicrobeStage.cs:528]: Might need to add a logic for automatic closing for the tutorial window after a certain interval.

File: locale/en.po
Comment (Kasterisk) [inline locale/en.po:5699]: Other people might have a better idea but I think the phrasing could still be improved. I suggest adding a reference to the daylight meter in the environment panel for one.

**Review by Kasterisk — CHANGES_REQUESTED** [review]

File: locale/en.po
Comment (hhyyrylainen) [reply locale/en.po:5699]: I'll also vote that the wording should be improved. For one the screen doesn't darken, but the environment does. The text can be made shorter by saying that photosynthesis is ineffective during the night. And the last part "when using those organelles" could be omitted. Instead these changes probably add enough space to refer to the environment panel saying the player can look there for the light level.

File: src/microbe_stage/MicrobeStage.cs
Comment (hhyyrylainen) [reply src/microbe_stage/MicrobeStage.cs:528]: I agree, and the timer should be in the tutorial class itself.

Comment (ender-titan1) [reply src/microbe_stage/MicrobeStage.cs:528]: Ok, I'll add that

File: src/microbe_stage/MicrobeStage.tscn
Comment (hhyyrylainen) [inline src/microbe_stage/MicrobeStage.tscn:2218]: Pause menu is unintentionally visible again.

File: src/microbe_stage/MicrobeStage.cs
Comment (hhyyrylainen) [inline src/microbe_stage/MicrobeStage.cs:523]: You should call GetCompoundAmount on the Patch type rather than boring into its properties here.

**Review by hhyyrylainen — CHANGES_REQUESTED** [review]
I wasn't going to do a full review, but I ended up doing one anyway with the separate comments I just posted.

File: locale/en.po
Comment (Kasterisk) [inline locale/en.po:6050]:
```suggestion
"Look out for drops in the sunlight meter on the environment panel.\n"
"\n"
"Nighttime comes as the environment darkens\n"
"Photosynthesis is ineffective during nighttime.\n"
"So make sure you have enough storage to survive the night."
```
I think this is now starting to sound good.

**Review by Kasterisk — CHANGES_REQUESTED** [review]

File: simulation_parameters/Constants.cs
Comment (Kasterisk) [inline simulation_parameters/Constants.cs:782]: I think this could be a little longer, maybe 20 or 25.

File: locale/en.po
Comment (Kasterisk) [inline locale/en.po:6048]:
```suggestion
"Nighttime comes as the environment darkens.\n"
```

**Review by Kasterisk — CHANGES_REQUESTED** [review]

**Review by Kasterisk — APPROVED** [review]

File: src/microbe_stage/MicrobeStage.cs
Comment (Kasterisk) [inline src/microbe_stage/MicrobeStage.cs:525]: One more thing, can you make `0.01f` a constant field in `Constants`?

**Review by Kasterisk — CHANGES_REQUESTED** [review]
Merge conflicts need to be fixed. Otherwise looks good to me for merging.

File: src/microbe_stage/MicrobeStage.tscn
Comment (hhyyrylainen) [inline src/microbe_stage/MicrobeStage.tscn:2218]: This is the opposite of the usual problem, so I guess we can allow this and see how long it is until Godot wants to automatically remove this...

File: src/microbe_stage/MicrobeStage.cs
Comment (hhyyrylainen) [reply src/microbe_stage/MicrobeStage.cs:525]: This would be nice to name. I think right now this is picked so that there is some effect during the night?

**Review by hhyyrylainen — CHANGES_REQUESTED** [review]
I think overall this is very good already but I'd like to see a few slight tweaks.

File: src/microbe_stage/MicrobeStage.cs
Comment (ender-titan1) [reply src/microbe_stage/MicrobeStage.cs:525]: I picked it so that the tutorial wouldn't show up in 1% light levels, as it is unlikely people will develop photosynthesis in those areas

**Review by hhyyrylainen — APPROVED** [review]
Code changes look fine to me now, and also in playtesting I didn't find problems.

**Comment by hhyyrylainen** [comment]
I'll merge this once the merge conflicts in translation files are fixed.

**Comment by hhyyrylainen** [comment]
I'll do the translations merge myself. I'll assume only English text was added for this feature to massively speed up the merge conflict resolution progress.

**Review by Kasterisk — APPROVED** [review]

**Pull request merged**
